stream/BidirectionalStream.java

```diff
@@ -48,11 +47,11 @@
     private final RequestTransmitter.Dispatcher dispatcher;
     private final AtomicBoolean isOpen;
 
-    public BidirectionalStream(Channel channel, RequestTransmitter transmitter) {
+    public BidirectionalStream(GraknStub.Core stub, RequestTransmitter transmitter) {
         resPartCollector = new ResponseCollector<>();
         resCollector = new ResponseCollector<>();
         isOpen = new AtomicBoolean(false);
-        dispatcher = transmitter.dispatcher(GraknGrpc.newStub(channel).transaction(new ResponseObserver()));
```

Comment:
We should be wary of this change and how it affects opening many parallel transactions - excessively reusing Stubs has caused stalling before in client-python. But I don't _think_ we've seen such issues in client-java before, just something to be aware for.